intel_extension_for_transformers/neural_chat/examples/talkingbot/front_end/.env

```diff
@@ -0,0 +1 @@
+AUDIO_URL = 'https://neuralstudio.eglb.intel.com/talkingbot'
```

Comment:
is this mandory?

Reply:
No, it only allows users to manually modify the location of the backend interface, and it has been set to empty to avoid misunderstanding.